vocabularies/PDF.xml

```diff
@@ -99,6 +99,12 @@
             <String>Max result size for PDF documents. Used if $top has been provided and $top > ResultSizeMaximum</String>
           </Annotation>
         </Property>
+        <Property Name="IANATimezoneFormat" Type="Edm.Boolean">
+          <Annotation Term="Core.Description" String="Formatting DateTimeOffset in IANA-time zone supported" />
```

Comment:
Don't understand what this means. Can you give an example of an `Edm.DateTimeOffset` value that is "formatted in a IANA-time zone"?

Is `DateTimeOffset` another property? Where does it come from?

Reply:
Hi Heiko,

here is the setup:
The timestamp contains value  "2017-04-20T13:37:00Z"
My time zone (IANA) is Europe/Berlin

The result in PDF will be "20.04.2017 15:37:00, 0000000 Europe/Berlin"

Let me know if you need further information or examples.

The description could maybe be a little bit more precise. What is actually meant is "Formatting properties with type EDM.DateTimeOffset in IANA-time zone supported". So maybe it is enough to add the "EDM." before "DateTimeOffset"? 

Best regards,

Florian

Reply:
Hi Heiko,

there are two possibilities:

1. By defaults time stamp fields shall be shown in the users time zone (SU3/SU01)
2. There shall also be a support for showing a field in a dedicated IANA time zone

I'll send you the link to our one-pager with more information.

Best regards,

Florian